File: CPP/Baju.cpp
/*Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
mengerjakan soal Tugas Praktikum 2 dalam mata kuliah DPBO 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin */
#include <iostream>
#include <string>
#include "Aksesoris.cpp"

using namespace std;

class Baju : public Aksesoris{//inheritance kelas Baju kepada kelas Aksesoris
    private://atribut kelas Baju
        string untuk;
        string size;
        string merk;
    
    public:
        Baju(){//konstruktor
            this->untuk = "";
            this->size = "";
            this->merk = "";
        }

        Baju(string id, string nama_produk, int harga_produk, int stok_produk, string jenis, string bahan, string warna, string untuk, string size, string merk) : Aksesoris(id, nama_produk, harga_produk, stok_produk, jenis, bahan, warna){
           this->untuk = untuk;
           this->size = size;
           this->merk = merk;
       }

       //getter dan setter
        string get_untuk(){
            return this->untuk;
        }

        void set_untuk(string untuk){
            this->untuk = untuk;
        }

        string get_size(){
            return this->size;
        }

        void set_size(string size){
            this->size = size;
        }

        string get_merk(){
            return this->merk;
        }

        void set_merk(string merk){
            this->merk = merk;
        }

        ~Baju(){//destruktor

        }
};

File: PHP/Baju.php
<!-- Saya Klara Ollivviera Augustine Gunawan dengan NIM 2306205 
mengerjakan soal Tugas Praktikum 1 dalam mata kuliah DPBO 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin -->

<?php

include 'Aksesoris.php';

class Baju extends Aksesoris{
    private $untuk = "";
    private $size = "";
    private $merk = "";

    public function __construct($id = "", $nama_produk = "", $harga_produk = 0, $stok_produk = 0, $foto_produk = "", $jenis = "", $bahan = "", $warna = "", $untuk = "", $size = "", $merk = ""){
        parent::__construct($id, $nama_produk, $harga_produk, $stok_produk, $foto_produk, $jenis, $bahan, $warna);
        $this->untuk = $untuk;
        $this->size = $size;
        $this->merk = $merk;
    }

    public function set_untuk($untuk){
        $this->untuk = $untuk;
    }

    public function get_untuk(){
        return $this->untuk;
    }

    public function set_size($size){
        $this->size = $size;
    }

    public function get_size(){
        return $this->size;
    }

    public function set_merk($merk){
        $this->merk = $merk;
    }

    public function get_merk(){
        return $this->merk;
    }
}
?>